modules/SpyUtils.js

```diff
@@ -4,6 +4,10 @@ import { isFunction } from './TestUtils'
 
 const noop = () => {}
 
+const engineSupportsConfigurableFunctionLength = (function () {
+  return Object.getOwnPropertyDescriptor(() => {}, 'length').configurable
```

Comment:
you're right, i'll change it to `typeof Object.getOwnPropertyDescriptor === 'function' && Object.getOwnPropertyDescriptor(() => {}, 'length').configurable`